src/redux/actions/dictionaries/dictionaryActionCreators.js

```diff
@@ -81,3 +82,14 @@ export const fetchDictionary = data => (dispatch) =>{
       dispatch(isFetching(false));
     });
   };
+
+export const fetchDictionaryConcepts = data => (dispatch) =>{
+  dispatch(isFetching(true));
+  return api.dictionaries
+    .fetchDictionaryConcepts(data)
+    .then(
+      (payload) =>{
+      dispatch(dictionaryConceptsIsSuccess(payload));
+      dispatch(isFetching(false));
+      });
+};
```

Comment:
Good job. However, I noticed there is no catch method for handling errors possibly returned from the server. Or are you handling the errors another way?

Reply:
No I am not handling errors any other ways. You are actually correct, I should be handling errors from the server. Thanks for noticing that, I will be implementing that feedback.

Reply:
@emmabaye I have resolved this and updated the PR.